utils/api/benefits/gisBenefit.ts

```diff
@@ -322,11 +322,9 @@ export class GisBenefit extends BaseBenefit<EntitlementResultGeneric> {
             this.translations.detailWithHeading.partnerEligibleButAnsweredNo
           )
         } else {
-          if (!this.input.invSeparated) {
```

Comment:
there is a case for which this was necessary but i cannot find it

Reply:
found it 
![image](https://github.com/DTS-STN/eligibility-estimator/assets/52539578/f9db21e6-e551-4a05-8db1-7c6bdba7c3d0)

Reply:
Good memory, I forgot about that duplicate scenario! Thanks